scripts/coverage.sh

```diff
@@ -14,13 +14,14 @@ reportName="lcov-${CI_COMMIT:0:9}"
 
 if [[ -n $1 ]]; then
   crate="--package $1"
+  shift
```

Comment:
To forward the rest to `cargo test` at https://github.com/solana-labs/solana/pull/6345/files#diff-c29225768574b8edd4f3608f595c29f7R41, `shift` `$1` here.